packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -1,11 +1,14 @@
 <template>
     <div>
-        <error-dialog
-            :is-open="shouldShowErrorDialog"
-            :error-code="hasNonFulfillableErrorCode"
-            :service-type="serviceType"
-            @close="handleErrorDialogClose"
-            @checkout-error-dialog-button-click="handleErrorDialogButtonClick" />
+        <component
+            :is="messageType.name"
+            v-if="message"
+            ref="errorAlert"
```

Comment:
We should probably change this ref now. It's no longer necessarily an `errorAlert`.